templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -528,11 +551,11 @@ contents:
 
       # Remove bridges created by openshift-sdn
       ovs-vsctl --timeout=30 --if-exists del-br br0
-      
-      # Recycle NM connections
-      reload_nm
 
       # Make sure everything is activated
+      for connection in $(nmcli -g NAME c | grep -- "$MANAGED_NM_CONN_SUFFIX"); do
+        activate_nm_conn "$connection"
+      done
```

Comment:
conflict: remove mentions of br-ex 1 and phys1 as 4.8 has no support for secondary bridge in configure-ovs.